### ISSUE TITLE
Needless scrollbars appear (w/ fix)

### ISSUE THREAD

**Issue by jflamy** [alejandro-du/crudui#44]
In version 3.7, there are extra scrollbars that appear when a toolbar is present because the grid is set to the full size of the parent, and so there is not enough room for the toolbar and the grid.

The fix is simple -- in WindowsBasedCrudLayout(), replace `mainComponentLayoutsetSizeFull()` with setWidth("100%").  The setExpand() a few lines below will do the rest (using all the vertical space)
```
        mainComponentLayout.setWidth("100%");
        mainComponentLayout.setMargin(false);
        mainComponentLayout.setPadding(false);
        mainLayout.add(mainComponentLayout);
        mainLayout.expand(mainComponentLayout);
```

**Comment by alejandro-du** [comment]
Thanks for reporting. Would you be able to do a PR with the fix?

**Comment by jflamy** [comment]
Done.  Learning github and fancy git/gitflow as I go.

**Comment by alejandro-du** [comment]
Merged and published(version 3.7.3). Thanks!
